zospy/analyses/polarization.py

```diff
@@ -19,7 +19,7 @@
 
 
 def _get_number_field(name: str, text: str) -> str:
-    return re.search(rf"{name}\s*:\s*(-?[\d{_config.DECIMAL_POINT}]+)", text).group(1)
+    return re.search(rf"{name}\s*:\s*(-?[\d{_config.DECIMAL_POINT}]+[Ee]?[+-]?\d{0,3})", text).group(1)

```

Comment:
For me, the this does not work, to make it work, the last part has to be updated to `\d{{0,3}}` to escape the curly brackets in the formatted raw string. 

However, as @crnh also indicated, I think the regex for the exponent can be updated. In addition, I also think we should maybe use `re.escape()` for both `name` and `_config.DECIMAL_POINT` to prevent including accidental regex patterns. Thus, I would like to change the regex pattern to:

```suggestion
    return re.search(rf"{re.escape(name)}\s*:\s*([-+]?\d+{re.escape(_config.DECIMAL_POINT)}?\d*(?:[Ee][-+]?\d+)?)", text).group(1)

```